### PR TITLE
 Added Merge function and tests

### PR DESCRIPTION
**Pull request by MDHowden2020**
Please review my code savage Sage and Anhtagonistic

### CODE REVIEW

File: tests/merge_sorted_lists_test.py [hunk 1]
```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
+def test_for_1_and_2():
+    # Arrange
+    l1 = [1]
+    l2 = [2]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
```
Comment (Coolcud) [inline tests/merge_sorted_lists_test.py:10]: Rename answer to a more accurate variable name.

**Review by Coolcud — COMMENTED** [review]
Good job....I guess

File: functions/merge_sorted_lists.py [hunk 2]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
```
Comment (hawktalk25) [inline functions/merge_sorted_lists.py:2]: Remove pass

File: functions/merge_sorted_lists.py [hunk 3]
```diff
@@ -1,2 +1,4 @@
...
+    merged = l1 + l2
+
+    return sorted(merged)
```
Comment (hawktalk25) [inline functions/merge_sorted_lists.py:4]: sorted() has a time complexity of O(n log n). May need to use a different sorting method for lower time complexity

**Review by hawktalk25 — COMMENTED** [review]
Tests look good! Nice work! :D

**Pull request merged**